code/__defines/explosion.dm

```diff
@@ -0,0 +1,15 @@
+// game.dm
+//We used to use linear regression to approximate the answer, but Mloc realized this was actually faster.
+//And lo and behold, it is, and it's more accurate to boot.
+/proc/cheap_hypotenuse(Ax,Ay,Bx,By)
+	return sqrt(abs(Ax - Bx)**2 + abs(Ay - By)**2) //A squared + B squared = C squared
```

Comment:
imagine not using defines for this
is there any disadvantage to using defines for small stuff like this, anyhow?

Reply:
Make it a define yeah

Reply:
its just a draft of port. but sure